R/util.R

```diff
@@ -9,7 +9,7 @@ traduire_file <- function(path) {
 
 
 read_file <- function(path) {
-  paste(readLines(path), collapse = "\n")
+  paste(readLines(path, encoding = "UTF-8"), collapse = "\n")
```

Comment:
This has found an issue already. Our test json has UTF-8 encoding - can we force this on users or do we have to be more flexible?

Reply:
json should always be utf-8. why are we reading with readLines and not into jsonlite directly?

Reply:
I think because i18next (or at least the call to addResourceBundle) is expecting a string of JSON but `jsonlite::read_json` will convert to R types, list and data frames

Reply:
ah of course. I think that the encoding option you've added is fine though